README.md

```diff
@@ -72,6 +72,7 @@ Parameters:
 * _Prefix to add to all generated filenames._ (`prefix`) : Mapshot will prefix all files it creates with that value. Factorio mods only allow writing within `script-output` subdirectory of Factorio data dir; the prefix is relative to that directory.
 * _Pixel size for generated tiles._ (`resolution`) : Size in pixels for the generated images. There is not a lot of reasons to change this value - if you want more or less details, change `tilemin`.
 * _Pixel size for generated tiles._ (`jpgquality`) : Compression quality for the generated image.
+* _Pixel size for generated tiles._ (`minjpgquality`) : Compression quality for the generated image when no player entities.
```

Comment:
nit: "when no player entities" -> "when no player entities are present. If set to 0, do not render a tile at all; instead, the map rendering will fallback to a lower zoom level as needed."